content/en/docs/tasks/extend-kubernetes/socks5-proxy-access-api.md

```diff
@@ -135,6 +121,24 @@ NAMESPACE     NAME                                     READY   STATUS      RESTA
 kube-system   coredns-85cb69466-klwq8                  1/1     Running     0          5m46s
 ```
 
+{{< notes >}}
+- Before kubectl 1.24, most kubectl commands worked when using a socks proxy, except `kubectl exec`.
+- kubectl supports both `HTTPS_PROXY` and `https_proxy` environment variables. These are used by other 
+  programs that support SOCKS, such as curl. Therefore in some cases it 
+  will be better to define the environment variable on the command line:
+```shell
+HTTPS_PROXY=socks5://localhost:1080 kubectl get pods
+```
+- When using `proxy-url`, the proxy is used only for the relevant kubectl context. Whereas the environment variable
+will affect all contexts. 
+- The k8s API server hostname can be further protected from DNS leakage by using the `socks5h` protocol name
+instead of the more commonly known `socks5` protocol shown above. In this case, kubectl will ask the proxy server
+(such as an ssh bastion) to resolve the k8s API server domain name, instead of resolving it on the system running
+`kubectl`. Note also that with `socks5h`, a k8s API server URL like `https://localhost:6443/api` does not refer 
+to your local client computer. Instead, it refers to `localhost` as known on the proxy server (eg the ssh bastion).
```

Comment:
```suggestion
  instead of the more commonly known `socks5` protocol shown above. In this case, kubectl will ask the proxy server
  (such as an ssh bastion) to resolve the k8s API server domain name, instead of resolving it on the system running
  `kubectl`. Note also that with `socks5h`, a k8s API server URL like `https://localhost:6443/api` does not refer 
  to your local client computer. Instead, it refers to `localhost` as known on the proxy server (eg the ssh bastion).
```